core/store/src/db.rs

```diff
@@ -547,9 +549,13 @@ impl RocksDBOptions {
     }
 
     /// Opens the database in read/write mode.
-    pub fn read_write<P: AsRef<std::path::Path>>(self, path: P) -> Result<RocksDB, DBError> {
+    pub fn read_write<P: AsRef<std::path::Path>>(
+        self,
+        path: P,
+        enable_statistics: bool,
```

Comment:
`RocksDBOptions` is a builder object, it exists so that we don’t have to add dozens of raw Boolean arguments. let’s add an 

   pub fn enable_statistics(mut self, yea: bool) -> Self


Builder method instead

Reply:
Good idea, I misunderstood the idea of `RocksDBOptions`.